generated/src/aws-cpp-sdk-dynamodb/source/DynamoDBClientConfiguration.cpp

```diff
@@ -0,0 +1,70 @@
+﻿/**
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+#include <aws/dynamodb/DynamoDBClientConfiguration.h>
+
+namespace Aws
+{
+namespace DynamoDB
+{
+
+
+bool IsEndpointDiscoveryEnabled(const Aws::String& endpointOverride, const Aws::String &profileName)
+{
+  bool enabled = false;
+  if (!endpointOverride.empty())
+  {
+    enabled = false;
+  }
+  else
+  {
+    static const char* AWS_ENABLE_ENDPOINT_DISCOVERY_ENV_KEY = "AWS_ENABLE_ENDPOINT_DISCOVERY";
+    static const char* AWS_ENABLE_ENDPOINT_DISCOVERY_PROFILE_KEY = "AWS_ENABLE_ENDPOINT_DISCOVERY";
+    static const char* AWS_ENABLE_ENDPOINT_ENABLED = "true";
+    static const char* AWS_ENABLE_ENDPOINT_DISABLED = "false";
+
+    Aws::String enableEndpointDiscovery = Client::ClientConfiguration::LoadConfigFromEnvOrProfile(
+        AWS_ENABLE_ENDPOINT_DISCOVERY_ENV_KEY, profileName, AWS_ENABLE_ENDPOINT_DISCOVERY_PROFILE_KEY,
+        {AWS_ENABLE_ENDPOINT_ENABLED, AWS_ENABLE_ENDPOINT_DISABLED}, AWS_ENABLE_ENDPOINT_ENABLED);
+
+    if (AWS_ENABLE_ENDPOINT_DISABLED == enableEndpointDiscovery)
+    {
+      // enabled by default unless explicitly disabled in ENV, profile config file, or programmatically later
```

Comment:
dont we want disabled by default unless explicitly enabled in profile or config. or maybe im misreading what this means

Reply:
I've just moved some code blocks around, the comment does not match the logic.
Thanks for noticing, let me update it.